src/Microsoft.Azure.AppConfiguration.AspNetCore/Microsoft.Azure.AppConfiguration.AspNetCore.csproj

```diff
@@ -21,7 +21,7 @@
   <!-- Nuget Package Version Settings -->
 
   <PropertyGroup>
-    <OfficialVersion>8.0.0-preview.3</OfficialVersion>
+    <OfficialVersion>8.1.0-preview</OfficialVersion>
```

Comment:
What are the changes being shipped for preview?

Reply:
If it's just that one change for allocation id, it seems like a bug fix - we should bump up patch version instead of minor.

Reply:
This PR https://github.com/Azure/AppConfiguration-DotnetProvider/pull/600

Reply:
Since we have not introduced any new public facing API, it should be a patch update. 
8.0.1-preview

Reply:
Fine with me. That PR is slightly public since it adds a new field on parsed FeatureFlags `FeatureFlag.Telemetry.Metadata.AllocationId`

Reply:
Discussed offline. We will bump up the minor version here.